### PR TITLE
Development branch

### PR DESCRIPTION
**Pull request by meme-es**
This branch was used to develop the project, it includes the next files: index.html, style.css, image files and complementary files inside a well-defined folder tree.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        <li>
+          <form action="#" class="search">
+            <i></i>
+            <input type="text" name="search">
+          </form>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <section class="main">
```
Comment (euqueme) [inline index.html]: [OPTIONAL] change the `<section>` tag to be the `<main>` tag

File: assets/css/style.css [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+}
+
+.list-inline li {
+  display: inline-block;
+}
+
+/* nav bar */
+
+.nav-container {
+  background: linear-gradient(to bottom, #8c8c8c 0%, #555 50%, #7c7c7c 52%, #a8a8a8 100%);
```
Comment (euqueme) [inline assets/css/style.css]: try with this gradient instead do you think it looks more like the original? :thinking: 

`background: linear-gradient(to bottom,gray 0%,#7c7c7c 50%,#666 51%,gray);`

**Review by euqueme — APPROVED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Congratulations @meme-es  :balloon:  :tada: :tada: 

### This project is just amazing :1st_place_medal: and I just have to approve it :clap: you followed the rules gracefully and there are really no major changes to make, I do have a couple of optional changes for you:

## Navbar Gradient
[OPTIONAL] I think the navbar is more on a grayscale maybe you can consider to change it a little to look even closer to the original.

## Building with Backgrounds

**Promo section**

[OPTIONAL] this project is all about building with backgrounds and gradients so consider to change all your ` `tags for `<div>` tags with the background image of the promos.

## Semantic HTML

[OPTIONAL] I Strongly encourage the use of the `<main>` tag for the dominant content of the `<body>` of a document

I have left comments in your code with my suggestions for the mentioned optional changes so feel free to try them in the inspector and keep them if you'd like to :arrow_double_down: 

### That's it! :raised_hands: 

### For more projects like this! :clinking_glasses: 

Happy Coding! :woman_technologist: 

PROJECT STATUS: APPROVED :green_apple:

File: index.html (hunk 1 above)
Comment (meme-es) [reply index.html]: Thank you, my team gave me the same suggestion on morning meeting, and it's already applied.

File: assets/css/style.css (hunk 2 above)
Comment (meme-es) [reply assets/css/style.css]: Yeah! It looks more like the original, thank you again.

**Pull request merged**